tango_simlib/tests/test_fandango_json_parser.py

```diff
@@ -0,0 +1,183 @@
+"""This module tests the fandango_json_parser script.
```

Comment:
New source files should include a copyright block, similar to other code in the repo.  This particular repo is open source with a BSD license, so we use that.  For example:
https://github.com/ska-sa/tango-simlib/blob/533e605/tango_simlib/utilities/sim_xmi_parser.py#L1-L5

Use the current year, when adding a new file.  Copyright starts from the year the file was first created.

Reply:
Done